packages/quantic/force-app/main/default/lwc/quanticFacet/quanticFacet.js

```diff
@@ -72,10 +91,24 @@ export default class QuanticFacet extends LightningElement {
   }
 
   get values() {
-    return this.state.values.map(v => ({
-      ...v,
-      checked: v.state === 'selected'
-    })) || [];
+    return (
+      this.state.values.map((v) => ({
+        ...v,
+        checked: v.state === 'selected',
```

Comment:
There was a lot of discussion around if we should call this `checked` or `selected` since it's called selected in the state.

You decide @fbeaupreCoveo :) `checked`, or `selected` ?

Reply:
😕  I'm good with keeping it as is, `checked` seems pretty clear to me even though it's selected in the state